legacy/account/account_cmk.ftl

```diff
@@ -61,14 +61,6 @@
             [#assign volumeEncryptionKmsKeyName = formatName("account", "cmk", "volume", "encrypt")]
             [#assign volumeEncryptionKmsKeyAliasId = formatDependentResourceId(AWS_CMK_ALIAS_RESOURCE_TYPE, volumeEncryptionKmsKeyId)]
 
-            [#-- Check that service linked role exists --]
```

Comment:
IS this part of some other change? Doesn't appear related to s3 replication?

Reply:
it was discovered as part of testing the deployment, I needed the CMK deployment to work in us-east-1 in order to setup the s3 replica bucket